src/services/ui/src/pages/welcome/index.tsx

```diff
@@ -15,11 +16,21 @@ export const loader = (queryClient: QueryClient) => {
     return queryClient.getQueryData(["user"]);
   };
 };
+type NavigateFunction = (path: string) => void;
+
 
 export const Welcome = () => {
+  const navigation=useNavigate();
+  const navigate: NavigateFunction = (path) => {
+    navigation(path);
+  };
+
+
+
+
+
```

Comment:
whats up with all these spaces and this formatting? youve gotta get your format on save working yo